lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -2386,6 +2389,72 @@ export class StandardController implements IController {
         );
         return res;
     }
+
+    private logMultipleInstances(
+        performanceEvent: InProgressPerformanceEvent,
+        request?: InitializeApplicationRequest
+    ): void {
+        const appId = request?.appId || this.config.auth.clientId;
+        // @ts-ignore
+        window.msal = window.msal || {};
+        // @ts-ignore
+        window.msal.appIds = window.msal.appIds || [];
+
+        // @ts-ignore
+        if (this.checkForSameClientId(appId, window.msal.appIds)) {
+            this.logger.warning(
+                "There is already an instance of MSAL.js in the window with the same client id."
+            );
+            // @ts-ignore
+        } else if (this.checkForMultipleInstances(appId, window.msal.appIds)) {
+            this.logger.warning(
+                "There is already an instance of MSAL.js in the window."
+            );
+        }
+
+        // @ts-ignore
+        if (!(window.msal.appIds.includes(appId) && appId.includes("."))) {
```

Comment:
nit: Just a little easier to understand IMO
```suggestion
        if (!window.msal.appIds.includes(appId) && !appId.includes(".")) {
```